llvm/lib/Target/AArch64/AArch64LoopIdiomTransform.cpp

```diff
@@ -592,7 +660,8 @@ Value *AArch64LoopIdiomTransform::expandFindMismatch(
   Value *SVERhsLoad = Builder.CreateMaskedLoad(SVELoadType, SVERhsGep, Align(1),
```

Comment:
Have you try to use normal load? And generate an epilogue loop after the SVE loop, I think there may be more performance impovement.